ui/lib/routing-tree.js

```diff
@@ -0,0 +1,260 @@
+// Setup
+
+// Modify the diameter to expand/contract space between nodes.
+var diameter = 1240;
+
+var tree = d3.layout.tree()
+    .size([360, diameter / 2 - 120])
+    .separation(function(a, b) { return (a.parent == b.parent ? 1 : 2) / a.depth; });
+
+var diagonal = d3.svg.diagonal.radial()
+    .projection(function(d) { return [d.y, d.x / 180 * Math.PI]; });
+
+var svg;
+
+var tooltip = d3.select("body")
+    .append("div")
+    .style("position", "absolute")
+    .style("background-color", "white")
+    .style("border", "1px solid #ddd")
+    .style("font", "9px monospace")
+    .style("padding", "4px 2px")
+    .style("z-index", "10")
+    .style("visibility", "hidden");
+
+// Global for now so we can play with it from the console
+function parseSearch(searchString) {
+  var labels = searchString.replace(/{|}|\"|\s/g, "").split(",");
```

Comment:
Yeah, that seems to be the case. It happens on my own repo with this code as well, even though the code compiles and runs correctly. I guess it's something about the syntax highlighter not picking up that it's within a regexp literal.